src/main/features/offer/form/models/offer.ts

```diff
@@ -0,0 +1,48 @@
+import { MaxLength, IsDefined, ValidateNested } from 'class-validator'
+import { Serializable } from 'models/serializable'
+import { IsNotBlank } from 'app/forms/validation/validators/isBlank'
+import { IsFutureDate } from 'forms/validation/validators/dateFutureConstraint'
+import { LocalDate } from 'forms/models/localDate'
+import { IsValidLocalDate } from 'forms/validation/validators/isValidLocalDate'
+
+export class ValidationErrors {
+  static readonly DATE_REQUIRED: string = 'Please enter a valid date'
+  static readonly DATE_NOT_VALID: string = 'Please enter a valid date'
+  static readonly FUTURE_DATE: string = 'Enter a offer date in the future'
+  static readonly OFFER_REQUIRED: string = "You haven't made your offer"
+  static readonly OFFER_TEXT_TOO_LONG: string = 'Enter offer no longer than $constraint1 characters'
```

Comment:
Would you mind confirming this with Ben and/or Ronan? As far as I remember the message we usually go for here is *You’ve entered too many characters*.

Reply:
this was in the story, anyway will confirm this.

Reply:
The limit is fine, I would just like to double-check the validation message with content team as it seems to be inconsistent with other similar validations.